### PR TITLE
MKMapView extensions

### PR DESCRIPTION
**Pull request by vadymmarkov**
There are multiple `MKMapView` extensions useful for clustering.

### CODE REVIEW

File: Sources/Extensions/MKMapView+PinFloyd.swift [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import MapKit
+
+// MARK: - Tile bounding box
+
+struct MapTileBoundingBox {
```
Comment (onmyway133) [inline Sources/Extensions/MKMapView+PinFloyd.swift]: Is this the same as this bounding box http://wiki.openstreetmap.org/wiki/Bounding_Box? which uses the same coordinate system as in `Cocoa`?

File: Sources/Extensions/MKMapView+PinFloyd.swift [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  let minX: Int
```
Comment (onmyway133) [inline Sources/Extensions/MKMapView+PinFloyd.swift]: What does minX mean? Does it mean `longitude`?

File: Sources/Extensions/MKMapView+PinFloyd.swift [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    case 13...15:
+      return 64
+    case 16...18:
+      return 32
+    case 19..<Int.max:
+      return 16
+    default:
+      return 88
+    }
+  }
```
Comment (zenangst) [inline Sources/Extensions/MKMapView+PinFloyd.swift:42]: Nifty 😁

File: Sources/Extensions/MKMapView+PinFloyd.swift (hunk 1 above)
Comment (vadymmarkov) [reply Sources/Extensions/MKMapView+PinFloyd.swift]: Not really, maybe it makes sense to rename it to something like `MapTile`.

**Review by zenangst — APPROVED** [review]
👍

File: Sources/Extensions/MKMapView+PinFloyd.swift (hunk 2 above)
Comment (vadymmarkov) [reply Sources/Extensions/MKMapView+PinFloyd.swift]: No, here we work with `MKMapRect` coordinates, not with latitude and longitude.

File: Sources/Extensions/MKMapView+PinFloyd.swift (hunk 1 above)
Comment (zenangst) [reply Sources/Extensions/MKMapView+PinFloyd.swift]: `MapTile` might make more sense 😎

**Comment by zenangst** [comment]
🚀

**Pull request merged**